src/modules/articles/components/extensions/Extension.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { IArticle } from '../../queries';
+import { extensions } from './extensions';
+
+interface IProps {
+    type: string,
+    props: string,
+    article: IArticle,
+    root: Element
+}
+
+const ExtensionHelper: React.FunctionComponent<IProps> = ({type, props, article}) => {
+    let propsObj;
+    try {
+        propsObj = JSON.parse(props);
+    }
+    catch(e) {
+        console.error(`Unable to parse props "${props}" in article extension of type ${type} (in Extension).`)
+        return null;
```

Comment:
Are you using --strict? Normally I don't think this would typecheck

Reply:
Strict is set to true so yes. Maybe inferring any is allowed in catch params because the type often doesn't matter like here.